3D-microservices-observability/node-js-app/beachshirt/controllers/shoppingController.js

```diff
@@ -0,0 +1,35 @@
+/**
+* Driver for Shopping service provides consumer facing APIs supporting activities like browsing
+* different styles of beachshirts, and ordering beachshirts.
+* @author Yogesh Prasad Kurmi (ykurmi@vmware.com)
+*/
+const request = require('request');
+
+module.exports = function(app, config, log){
+
+    app.get('/shop/menu', function(req, res){
+        request.get("http://" + config.styling.host + ":" + config.styling.port + "/style",
```

Comment:
Would be better to use template strings here.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Might as well go the extra mile and make the protocol configurable.

Reply:
Probably should be a complete base URL